lib/base/application.cpp

```diff
@@ -171,7 +171,7 @@ void Application::SetResourceLimits(void)
 	rlimit rl;
 
 #	ifdef RLIMIT_NOFILE
-	rl.rlim_cur = 16 * 1024;
+	rl.rlim_cur = 64 * 1024;
```

Comment:
It would be interesting to control this via a env variable.

```c
#define DEFAULT_LIMIT 16
int limit;
limit = getenv('RESOURCES_LIMIT') ? atoi(getenv('RESOURCES_LIMIT')) : DEFAULT_LIMIT;
rl.rlim_cur = limit * 1024;
```